app/views/projects/show.html.erb

```diff
@@ -40,7 +44,11 @@ Project Details:
 </div>
 
 <div class="details">
-  <h2>Project Description</h2>
+  <h2>Project Description
+    <% if @project.metadata[:status] == @approve_status && @eligible_editor %>
```

Comment:
@JaymeeH I'm not going to hold up the PR, but another time you could have put this logic in the controller.  Logic in the view is a lot harder to test.  I might have added `@project_eligble_to_edit` flag in the controller and then used it in the two places in the view.

Reply:
+1 to put this in the controller. There are going to be lots of places where we will need to test whether the project is eligible to edit, and the criteria for that will likely evolve over time. Let's put the logic in a single place where it can be tested.